hs_core/irods.py

```diff
@@ -140,7 +148,8 @@ def create_ticket(self, user, path=None, write=False, allowed_uses=1):
         formatted = timeout.strftime("%Y-%m-%d.%H:%M")
         istorage.session.run('iticket', None, 'mod', ticket,
                              'expires', formatted)
-        return ticket
+
+        return ticket, self.irods_full_path(path)
```

Comment:
@alvacouch Since line 132 already formed the full path to assign to ```path```, it seems ```path``` can be directly returned without calling ```self.irods_full_path(path)```